src/jaxsim/mujoco/loaders.py

```diff
@@ -412,8 +430,8 @@ def convert(
             "camera",
             name="track",
             mode="trackcom",
-            pos="1 0 5",
-            zaxis="0 0 1",
+            pos="1.930 -2.279 0.556",
+            xyaxes="0.771 0.637 0.000 -0.116 0.140 0.983",
```

Comment:
Is this a leftover?

Reply:
No, it's intended to improve the default camera position

Reply:
Ok, how did you come up with these numbers? It could be useful to add a comment so that future users passing by do not get puzzled with them in case they need to be updated.

Reply:
I used `python -m mujoco.viewer --mjcf=<PATH_TO_URDF>` and manually adjusted the camera. I'll add a comment about this

Reply:
Addressed in https://github.com/ami-iit/jaxsim/pull/125/commits/3ac3f12cadc73bcbaac0afaab6c8fee150c58f92